mono/mini/mini.h

```diff
@@ -1349,6 +1349,8 @@ typedef struct {
 
 	MonoGSharedVtMethodInfo *gsharedvt_info;
 
+	MonoMemoryManager *mem_manager;
```

Comment:
Similar questions to adding it in the transform, but again if you're sure it's safe it makes for a very nice diff.